dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -31,8 +31,9 @@ internal class RumActionScope(
     inactivityThresholdMs: Long = ACTION_INACTIVITY_MS,
     maxDurationMs: Long = ACTION_MAX_DURATION_MS,
     private val rumEventSourceProvider: RumEventSourceProvider,
-    private val androidInfoProvider: AndroidInfoProvider
-) : RumScope {
+    private val androidInfoProvider: AndroidInfoProvider,
+    val trackFrustrations: Boolean
```

Comment:
can be made `private`, it seems it is not accessed from outside

```suggestion
    private val trackFrustrations: Boolean
```